e2e/config.js

```diff
@@ -63,6 +63,7 @@ module.exports = {
     amendInternationalElement: 'International element',
     amendOtherProceedings: 'Other proceedings',
     amendAttendingHearing: 'Attending the hearing',
+    addHearingDetails: 'Add hearing details',
```

Comment:
Should this not be `addHearingBookingDetails`?

Reply:
agreed 👍